packages/runtime/src/dataViews/content/RequestItemDVOs.ts

```diff
@@ -37,6 +38,12 @@ export interface CreateAttributeRequestItemDVO extends RequestItemDVO {
     sourceAttributeId?: string;
 }
 
+export interface DeleteAttributeRequestItemDVO extends RequestItemDVO {
+    type: "DeleteAttributeRequestItemDVO";
+    attributeId: string;
+    attribute: LocalAttributeDVO;
```

Comment:
The other RequestItemDVOs (for example the ShareAttributeRequestItemDVO) use DraftIdentityAttributeDVO and/or DraftRelationshipAttributeDVO at this point.

Reply:
Ahh that sounds plausible, and would then also explain the "Draft" in the names DraftIdentityAttributeDVO and DraftRelationshipAttributeDVO! 👍

Reply:
@Milena-Czierlinski absolutely correct.